src/test/resources/jobsInJenkinsFolder.json

```diff
@@ -0,0 +1,10 @@
+{
+  "_class" : "com.cloudbees.hudson.plugins.folder.Folder",
+  "jobs" : [
+    {
+      "_class" : "hudson.model.FreeStyleProject",
+      "name" : "Test Project",
+      "url": "http://localhost:8080/job/username"
```

Comment:
Just noticed PR got merged and feature ticket closed already but one part of the Job that Jenkins returns that we missed here is the `color` json attiribute, e.g.

`  "jobs": [
    {
      "_class": "org.jenkinsci.plugins.workflow.job.WorkflowJob",
      "name": "myPipelineJobBranchName",
      "url": "https://myJenkinsHost.com/job/folder1/job/folder2/job/myPipelineJob/job/myPipelineJobBranchName/",
      "color": "blue"
    },`

it's an attribute that could be used to determine if the job is currently running or not. e.g. the enumerated values I've discovered for this attribute are:

`*   - blue: completed/non-running, latest build passed
*   - blue_anime: running, latest build passed
*   - red: completed/non-running, latest build failed
*   - red_anime: running, latest build failed
*   - notbuilt: currently not running, pipeline branch has never run
*   - notbuilt_anime: currently running, pipeline branch has never run`

Reply:
Actually, my bad, I was mixing up the terms "pipeline" and "job". I see what I need now when passing job name in (from the new JobList endpoint)

`client.api().jobsApi().jobInfo("folder1/folder2/myPipeline", "pipelineBranchJobName");`

Reply:
> Just noticed PR got merged and feature ticket closed already but one part of the Job that Jenkins returns that we missed here is the `color` json attiribute, e.g.
> 
> ` "jobs": [ { "_class": "org.jenkinsci.plugins.workflow.job.WorkflowJob", "name": "myPipelineJobBranchName", "url": "https://myJenkinsHost.com/job/folder1/job/folder2/job/myPipelineJob/job/myPipelineJobBranchName/", "color": "blue" },`
> 
> it's an attribute that could be used to determine if the job is currently running or not. e.g. the enumerated values I've discovered for this attribute are:
> 
> * blue: completed/non-running, latest build passed
> * blue_anime: running, latest build passed
> * red: completed/non-running, latest build failed
> * red_anime: running, latest build failed
> * notbuilt: currently not running, pipeline branch has never run
> * notbuilt_anime: currently running, pipeline branch has never run

I think you have a point that we could add "color" to the "Jobs" object, so that I can get an overview of the status of pipeline/Job when I query JobList, without making another API call.
There's also a "disabled" for disabled jobs
      "color": "disabled" 

Could be another PR I suppose?

Reply:
Sure could!!!